Practice3/Practice3.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Practice3
+{
+    class Task
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("***Practice 3***");
+            Console.Write("\nCalculate sum of all integer values of the min-max range those are divided on 3 and not divided on 5.");
+            Console.Write("\n\nPlease choose a decision:");
+            Console.Write("\n1. User can add as many values as s(he) wants.");
+            Console.Write("\n2. User defines min and max values of the range only.");
+            Console.Write("\nOption... ");
+
+            int input = Convert.ToInt16(Console.ReadLine());
```

Comment:
could you please fix this exception? (happens when the user inputs a string)

![image](https://user-images.githubusercontent.com/9382934/113508417-9090c700-9558-11eb-81c4-52039980d9fb.png)

In addition, when I enter e.g. "5", the program is just closed.
Could you please handle this as well?

Reply:
There's an exception when max value is less than min

![image](https://user-images.githubusercontent.com/9382934/113509361-cdab8800-955d-11eb-84ed-c89114a63d69.png)

AND there's no validation for negative values despite they are not natural so should not be accepted

Reply:
Done